autometa/taxonomy/majority_vote.py

```diff
@@ -243,17 +243,13 @@ def majority_vote(fasta, ncbi_dir, outdir, votes_fname, lca_fname=None, **kwargs
         <lca.tsv filename> (the default is </path/to/`outdir`/`fasta`.lca.tsv).
     **kwargs : dict
         Further parameters that may be passed along to LCA, LCA.blast2lca and
-        rank_taxids as dicts of key-value pairs. Note: Types must match.
-        Defaults:
-        -LCA
-            usepickle=True (bool)
-            verbose=True (bool)
-        -LCA.blast2lca
-            blast=None (str)
-            hits_fpath=None (str)
-            force=False (bool)
-        -rank_taxids
-            verbose=False (bool)
+        rank_taxids as dicts of key-value pairs.
+        Note: Types must match.
+        Defaults are listed below:
+        * LCA:  usepickle=True (bool), verbose=True (bool)
+        * LCA.blast2lca: blast=None (str), hits_fpath=None (str), force=False (bool)
+        * rank_taxids: verbose=False (bool)
+
```

Comment:
Again all of it is outputting in a single paragraph. 
I think you need a blank line before the bullets.
https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/basics.html#lists-and-quote-like-blocks